src/App.js

```diff
@@ -1,15 +1,26 @@
 import React from "react";
-
+import Heading from "./Heading";
 import Bookings from "./Bookings";
+import TouristInfoCards from "./TouristInfoCards";
+import Restaurant from  "./Restaurant"
+import SearchResults from "./SearchResults";
+import Footer from "./Footer";
 import "./App.css";
+import Order from "./Order";
 
 const App = () => {
+
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <header className="header">CYF Hotel</header>
+      <Heading />
       <Bookings />
+      <TouristInfoCards />
+      <Restaurant/>
+      <SearchResults />
+      <Order/>
+      <Footer />
     </div>
   );
 };
-
-export default App;
+export default App();
```

Comment:
Careful with this - you _want_ to export the whole function, `App`, whereas here you're only exporting the _return_ of calling `App()`. When writing components, React will handle the calling of the function for you when you use the `<Component />` syntax.